File: AEDS 1/LISTA EXERCICIOS/LISTA 03 - MODULARIZAÇÃO/exercicio8.c
#include <stdio.h>
#include <stdlib.h> 
#include <math.h>
float calcula(int N)
{
    //Inicializar S. 
    float S = 0; 

    //Verifica de onde começa até onde vai. 
    for (int cont = 1; cont <= N; cont++)
    {
         S += (pow(N, 2) + 1) / (N + 3);
    }

    return S; 
}
int main()
{
    //Inicializar Variável. 
    int N; 

    //Receber valor de N. 
    printf("Informe u valor inteiro e positivo para N: "); 
    scanf("%d", &N); 

    printf("\nS = %.2f", calcula(N)); 

    return 0; 
}

File: AEDS 2/EXERCICIOS BASICOS/EXERCICIOS BÁSICOS DE JAVA/Exercicio8.java
//Exercicio para verificar se um número é positivo ou negativo. (Uso do If-ELse em java).
import java.util.Scanner; //Biblioteca para ler a entrada do usuário.

//Inicializar a classe com o mesmo nome que foi salvo no arquivo.
 class Exercicio8 {
    public static void main (String[] args) { //Método principal onde o programa começa a execução.
        //Inicializar variáveis.
        int num; 
        Scanner sc = new Scanner(System.in); //Criar um objeto Scanner para ler a entrada do usuário. (como se fosse um scanf em C).

        //Ler um número inteiro do usuário e armazená-lo na variável num.
        System.out.print("Digite um numero: "); 
        num = sc.nextInt(); //O próximo inteiro informado pelo usuário será armazenado em num.

        //Verificar se o número é positivo ou negativo usando uma estrutura de controle if-else e imprimir o resultado.
        if (num < 0) {
            System.out.println("O numero "+num+" e negativo.");
        }
        else {
            if (num == 0) {
                System.out.println("O numero "+num+" e zero.");
            }
            else {  
            System.out.println("O numero "+num+" e positivo.");
            }
        }

        //Fechar o scanner para evitar vazamento de memória.
        sc.close();
    }
}
